Library/OcCpuLib/OcCpuLib.c

```diff
@@ -1246,14 +1251,17 @@ ScanAmdProcessor (
       Cpu->MaxBusRatio
       ));
 
-    //
-    // CPUPM is not supported on AMD, meaning the current
-    // and minimum bus ratio are equal to the maximum bus ratio
-    //
-    Cpu->CurBusRatio = Cpu->MaxBusRatio;
-    Cpu->MinBusRatio = Cpu->MaxBusRatio;
+    // if under visualization, we already have those info.
```

Comment:
```
//
// When under virtualization this information is already available to us.
//
```

And please fix the indentation below, we use 2 spaces, not 4 according to EDK II code style.